src/components/AccordionPanel/AccordionPanel.tsx

```diff
@@ -62,8 +62,12 @@ export const AccordionPanel = ({
   const panelRef = useRef() as MutableRefObject<HTMLDivElement>;
   const [heightVar, setHeight] = useState('0');
   const [isActiveVar, setIsActive] = useState(isActive ? true : false);
-  const [ariaControlsVar, setAriaControls] = useState();
-  const [ariaLabelledByVar, setAriaLabelledBy] = useState();
+
+  const generatedAriaControlsId = useUID();
+  const ariaControlsVar = ariaControls || generatedAriaControlsId;
```

Comment:
`useUID` cannot be used in conditionals or hooks (to ensure consistent ID generation)